pkgs/applications/office/autokey/default.nix

```diff
@@ -28,15 +30,24 @@ python3Packages.buildPythonApplication rec {
     gtksourceview3
     libappindicator-gtk3
     libnotify
+    gnome.zenity
+    wmctrl
   ];
 
   propagatedBuildInputs = with python3Packages; [
     dbus-python
     pyinotify
     xlib
     pygobject3
+    packaging
   ];
 
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=(''${gappsWrapperArgs[@]} --prefix PATH : ${ lib.makeBinPath buildInputs })
```

Comment:
Don't wrap with all buildInputs to avoid unnecessary runtime deps

Reply:
Gotcha.